st2actions/st2actions/query/mistral/v2.py

```diff
@@ -42,13 +42,25 @@ def query(self, execution_id, query_context):
             status, output = self._get_workflow_result(exec_id)
             if output and 'tasks' in output:
                 LOG.warn('Key conflict with tasks in the workflow output.')
+        except requests.exceptions.ConnectionError:
+            msg = 'Unable to connect to mistral.'
```

Comment:
We should probably also include actual exception message and potentially also a traceback in the `msg` attribute since that's what we will store in the database.